### DISCUSSION TITLE
Disable shaders in a dimension

### DISCUSSION THREAD

**Discussion by blockninja124** [IrisShaders/Iris#2466] [Q&A]
I'm helping to make a space mod that has its own space dimension, but we've run into a problem getting it to work with shaders. The main problem is that after trying a bunch of shaders (about 5), we found that some of our custom rendering is being ignored/made invisible by the shaders. To fix this, I was wondering if it is possible to disable shaders in a particular dimension(s). I noticed that from [here](https://github.com/IrisShaders/ShaderDoc/blob/master/iris-features.md#dimension-folders) in the iris docs, you can have custom shaders per dimension. I was wondering if
A) I can have the mod change all active shader packs to include a file inside them that makes it so that dimension(s) doesn't have shaders
or
B) Disable shaders in a particular dimension in a different/easier way

**Comment by 3dgoose** [comment]
Did you find a way?
